.github/workflows/release.yml

```diff
@@ -511,22 +511,72 @@ jobs:
 
   trigger-docs-update:
     if: ${{ inputs.skip-release-notes != true }}
-    needs: push-tags
+    needs:
+      - push-tags
+      - check-version
     runs-on: ubuntu-22.04
     timeout-minutes: 10
+    permissions:
+      contents: write
+      pull-requests: write
+    env:
+      NEW_OSS_VERSION: ${{ needs.check-version.outputs.oss }}
+      NEW_EE_VERSION: ${{ needs.check-version.outputs.ee }}
+      GH_TOKEN: ${{ secrets.METABASE_AUTOMATION_USER_TOKEN }}
     steps:
       - uses: actions/github-script@v7
         with:
-          github-token: ${{ secrets.DOCS_UPDATE_TOKEN }}
-          script: |
-            github.rest.repos.createDispatchEvent({
+          github-token: ${{ secrets.METABASE_AUTOMATION_USER_TOKEN }}
+          script: | # js
+            await github.rest.repos.createDispatchEvent({
               owner: '${{ github.repository_owner }}',
               repo: '${{ secrets.DOCS_REPO }}',
               event_type: 'trigger-docs-update',
               client_payload: {
                 version: '${{ inputs.version }}'
               }
             });
+      - uses: actions/checkout@v4
+      - name: update releases.md
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.METABASE_AUTOMATION_USER_TOKEN }}
+          script: | # js
+            const fs = require('fs');
+
+            const releaseText = fs.readFileSync('./docs/releases.md', 'utf8');
+
+            const newReleaseText = releaseText
+              .replace(
+                /## Metabase Enterprise Edition releases\n\n/,
+                `## Metabase Enterprise Edition releases\n\n- [${process.env.NEW_EE_VERSION}](https://github.com/metabase/metabase/releases/tag/${process.env.NEW_EE_VERSION})`
+              )
+              .replace(
+                /## Metabase Open Source Edition releases\n\n/,
+                `## Metabase Open Source Edition releases\n\n- [${process.env.NEW_OSS_VERSION}](https://github.com/metabase/metabase/releases/tag/${process.env.NEW_OSS_VERSION})`
+              );
+
+            fs.writeFileSync('./docs/releases.md', newReleaseText);
+
+      - name: commit and push updates
+        run: | # bash
+          git config --global user.email "metabase-bot@metabase.com"
+          git config --global user.name "Metabase bot"
+
+          git checkout -b update-version-list-$NEW_OSS_VERSION
+          cd docs/
+
+          git add releases.md
+          git commit -m "Update release version list to include $NEW_OSS_VERSION"
+          git push origin update-version-list-$NEW_OSS_VERSION
+
+      - name: open pr with updates
+        run: | # bash
+          gh pr create --fill \
+            --label "Type:Documentation" \
+            --label "backport" \
+            --reviewer "jeff-bruemmer" \
+            --assignee "jeff-bruemmer"
```

Comment:
should we have a writers github group?

Reply:
Probably, but let's worry about that in a future PR.